docs/helm-chart/index.rst

```diff
@@ -140,3 +140,18 @@ will not start as the migrations will not be run:
 This is so these CI/CD services can perform updates without issues and preserve the immutability of Kubernetes Job manifests.
 
 This also applies if you install the chart using ``--wait`` in your ``helm install`` command.
+
+.. note::
+    While deploying this Helm chart with Argo, you might encounter issues with database migrations not running automatically on upgrade.
+
+To ensure database migrations with Argo CD, you will need to add:
+
+.. code-block::yaml
+
+    migrateDatabaseJob:
+        jobAnnotations:
+            "argocd.argoproj.io/hook": Sync
+
+This will ensure database migrations run when the Airflow Docker image is upgraded. This approach has a limitation in that the database migrations will run every time there is a ``Sync`` event in Argo. This is a trade-off for automation at the cost of some computational loss.
```

Comment:
```suggestion
This will run database migrations when the Airflow Docker image is upgraded. This approach has a limitation in that the database migrations will run every time there is a ``Sync`` event in Argo. This is a trade-off for automation at the cost of some computational loss.
```

Avoid double usage of "ensure database migrations", reads a bit nicer.